extensions/amp-autocomplete/0.1/test/test-amp-autocomplete.js

```diff
@@ -232,6 +232,52 @@ describes.realWin(
       });
     });
 
+    it('getRemoteData_() should proxy XHR to viewer', () => {
+      impl.isSsr_ = true;
+      impl.element.setAttribute('src', '');
+      const html =
+        '<div> <div>apple</div> <div>mango</div> <div>pear</div> </div>';
+      sandbox
+        .stub(impl.ssrTemplateHelper_, 'ssr')
+        .returns(Promise.resolve({html}));
+      return impl.getRemoteData_().then(() => {
+        expect(impl.ssrTemplateHelper_.ssr).to.be.calledOnce;
+      });
+    });
+
+    it('renderResults() should delegate template rendering to viewer', () => {
+      impl.hasTemplate_ = true;
+      const items = ['apple', 'mango', 'pear'];
+      const html =
+        '<div> <div>apple</div> <div>mango</div> <div>pear</div> </div>';
+      const sourceData = {html};
+      const rendered = doc.createElement('div');
+      items.forEach(item => {
```

Comment:
I think this is less clear than just creating the rendered container directly:

```javascript
const html = htmlFor(doc);
const rendered = html`
<div>
  <div data-value="apple"></div>
  <div data-value="mango"></div>
  <div data-value="pear"></div>
</div>
`;
```

Alternatively consider creating a helper function for rendering these for the various tests.